src/librustc/ty/instance.rs

```diff
@@ -54,7 +54,10 @@ pub enum InstanceDef<'tcx> {
         call_once: DefId,
     },
 
-    /// `drop_in_place::<T>; None` for empty drop glue.
+    /// `real_drop_in_place::<T>`.
+    /// The `DefId` is for `real_drop_in_place`.
```

Comment:
`real_drop_in_place` is gone (https://github.com/rust-lang/rust/pull/68335), it's just `drop_in_place` now.